modules/websearch/lib/websearch_templates.py

```diff
@@ -167,6 +168,9 @@ class Template:
         'wl': (int, CFG_WEBSEARCH_WILDCARD_LIMIT),
         }
 
+    if CFG_WEBSEARCH_OBELIX_REDIS:
+        search_results_default_urlargd['rm'] = (str, "rrm")
```

Comment:
This is causing some problem in other parts of Invenio, i.e. WebComment or InvenioConnector.

Maybe we should leave it as before and set it as default in the places we want, WDYT?